### PR TITLE
Add 'Training' button to home screen

### PR DESCRIPTION
**Pull request by amstone326**
_Spec_: https://docs.google.com/document/d/1nF6k9HG2Q9Xs-84u4pPC5QZBLlFVIp-qBmkGzkjeUwI/edit

cross-request: https://github.com/dimagi/commcare-core/pull/775

**Product Note**: Adding functionality requested by ICDS that allows app builders to designate app menus as "training menus", and have them show up under a separate home button labeled "Training".

_Screenshots_:

The new home button (only present if the app has a training module defined):
screenshot_20180316-163005

The new drawer item in the modern home screen UI (only present if the app has a training module defined):
screenshot_20180305-171039

What you see after selecting the training button:
screenshot_20180305-171048

### CODE REVIEW

File: app/src/org/commcare/activities/RootMenuHomeActivity.java [hunk 1]
```diff
@@ -38,7 +38,6 @@ public void onCreateSessionSafe(Bundle savedInstanceState) {
     @Override
     public void onResumeSessionSafe() {
         super.onResumeSessionSafe();
-        navDrawerController.reopenDrawerIfNeeded();
```
Comment (amstone326) [inline app/src/org/commcare/activities/RootMenuHomeActivity.java]: not related to this PR but I realized this was causing undesirable behavior while testing

**Comment by amstone326** [comment]
Screenshots:

The new home button (only present if the app has a training module defined). This needs a bit of scaling work that I'm having a lot of trouble with due to the vector drawable:
screenshot_20180305-171020

The new drawer item in the modern home screen UI (only present if the app has a training module defined):
screenshot_20180305-171039

What you see after selecting the training button:
screenshot_20180305-171048

**Comment by amstone326** [comment]
@ctsims See new screenshot! 😄

**Comment by ctsims** [comment]
@amstone326 Niiiiiice. All of our stock photos are going to look.... so much less terrible now. We've gotta get this in before the next big press release!

I'm wondering whether we should run the word "training" and the icon by one or two people from GS and get general feedback about the vocabulary? I guess people can translate on their own for individual projects, and I can't think of a better word (Self directed learning???) but a few people on the team have mentioned the idea, so one of them might have a brilliant idea.

**Comment by amstone326** [comment]
@ctsims I think the fact that it's translatable makes me think it can wait, but if you have a set of people in mind to ask, I'm happy to do so.

**Pull request merged**